File: beginner/java/2702.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int[] atendidos = new int[3];
        int[] requisitados = new int[3];
        int naoAtendidos = 0;

        atendidos[0] = scanner.nextInt();
        atendidos[1] = scanner.nextInt();
        atendidos[2] = scanner.nextInt();

        requisitados[0] = scanner.nextInt();
        requisitados[1] = scanner.nextInt();
        requisitados[2] = scanner.nextInt();

        for (int i = 0; i < 3; i++) {
            if (requisitados[i] > atendidos[i]) {
                naoAtendidos += requisitados[i] - atendidos[i];
            }
        }

        System.out.println(naoAtendidos);
    }
}

File: beginner/c/2702.c
#include <stdio.h>

int main() {
    int atendidos[3];
    int requisitados[3];
    int nao_atendidos = 0;

    scanf("%d %d %d", &atendidos[0], &atendidos[1], &atendidos[2]);
    scanf("%d %d %d", &requisitados[0], &requisitados[1], &requisitados[2]);

    for (int i = 0; i < 3; i++) {
        if (requisitados[i] > atendidos[i]) {
            nao_atendidos += requisitados[i] - atendidos[i];
        }
    }

    printf("%d\n", nao_atendidos);

    return 0;
}
